dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/tests/SymbolicNameTest.java

```diff
@@ -53,10 +53,15 @@ public void testSymbolicName() {
 	 */
 	@Test
 	public void testSingleton() {
+        StringBuilder errorMessage = new StringBuilder();
```

Comment:
Indentation ... tabs vs spaces, I can't tell?

Reply:
It is a mix in this class.  Mostly it is spaces.  I can make it all spaces if you would like.

Reply:
That would be my preference

Reply:
Spacing updated in the class.